<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="2" column="0">
    <widget class="QListWidget" name="listWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>180</height>
      </size>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="layoutMode">
      <enum>QListView::SinglePass</enum>
     </property>
     <property name="gridSize">
      <size>
       <width>220</width>
       <height>190</height>
      </size>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
     <property name="uniformItemSizes">
      <bool>true</bool>
     </property>
     <property name="selectionRectVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QComboBox" name="plotType">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Histogram</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Half Life</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="newPlotFrame">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QComboBox" name="fitType">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>No Fit</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Exponential Decay</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="errorBars">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Error Bars</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="plotLog">
          <property name="text">
           <string>Log</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Filter files:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pattern">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="class" stdset="0">
      <string>deep</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="horizontalSpacing">
       <number>2</number>
      </property>
      <property name="verticalSpacing">
       <number>1</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="2">
       <widget class="QPushButton" name="dirChange">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Select Directory</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/control/folder.png</normaloff>:/control/folder.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="5">
       <widget class="QLabel" name="pathLabel">
        <property name="text">
         <string>Directory:</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="fileOpenButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Open HISTOGRAM</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/control/document-open.png</normaloff>:/control/document-open.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="dirChange_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Load Project</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/control/projectfolder.svg</normaloff>:/control/projectfolder.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="fileOpenButton_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Open LIST DATA</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/control/document-open.png</normaloff>:/control/document-open.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QTextBrowser" name="comments">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="openLinks">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>dirChange</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>changeDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>415</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>712</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plotType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>plotTypeChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>456</y>
    </hint>
    <hint type="destinationlabel">
     <x>32</x>
     <y>492</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>Form</receiver>
   <slot>itemClicked(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>-14</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>Form</receiver>
   <slot>itemDoubleClicked(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>-20</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pattern</sender>
   <signal>returnPressed()</signal>
   <receiver>Form</receiver>
   <slot>patternChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>573</x>
     <y>456</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>492</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileOpenButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>browseFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>-19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dirChange_2</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>loadProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>532</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>-24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileOpenButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>browseListFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>-5</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeDirectory()</slot>
  <slot>itemClicked(QListWidgetItem*)</slot>
  <slot>plotMultiple()</slot>
  <slot>patternChanged()</slot>
  <slot>plotTypeChanged(int)</slot>
  <slot>openInNewWindow(bool)</slot>
  <slot>browseFile()</slot>
  <slot>itemDoubleClicked(QListWidgetItem*)</slot>
  <slot>loadProject()</slot>
  <slot>browseListFile()</slot>
 </slots>
</ui>
